### PR TITLE
Fix cursor behavior on AppContentListItem

### PR DESCRIPTION
**Pull request by ma12-co**
partially fix: #2403

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/Navigation/ConversationsList/AppContentListItem/AppContentListItem.vue [hunk 1]
```diff
@@ -148,6 +148,11 @@ export default {
 	justify-content: space-between;
 	align-items: center;
 	flex: 0 0 auto;
+	cursor: pointer;
+	// Make the cursor a ponter on every child element
+	* {
```
Comment (skjnldsv) [inline src/components/Navigation/ConversationsList/AppContentListItem/AppContentListItem.vue]: Hum, no.
This is usually when shit happens :p
Please properly target the needed entries

File: src/components/Navigation/ConversationsList/AppContentListItem/AppContentListItem.vue [hunk 2]
```diff
@@ -148,6 +148,7 @@ export default {
 	justify-content: space-between;
 	align-items: center;
 	flex: 0 0 auto;
+	cursor: pointer;
```
Comment (nickvergessen) [inline src/components/Navigation/ConversationsList/AppContentListItem/AppContentListItem.vue]: Duplicate of line 145

**Comment by nickvergessen** [comment]
This does not fix the flickering on the ActionText components for me :confused:

File: src/components/Navigation/ConversationsList/AppContentListItem/AppContentListItem.vue (hunk 2 above)
Comment (ma12-co) [reply src/components/Navigation/ConversationsList/AppContentListItem/AppContentListItem.vue]: :man_facepalming:

**Comment by ma12-co** [comment]
> This does not fix the flickering on the ActionText components for me confused

I think that this has is an `ActionText` issue

**Comment by nickvergessen** [comment]
But what is this here then about? I don't know any other related issue that would need solving

**Comment by ma12-co** [comment]
> But what is this here then about? I don't know any other related issue that would need solving

Basically just to make the cursor a pointer on AppContentListItem.

**Comment by nickvergessen** [comment]
Just rebased due to "conflicts" but no interaction was needed, so merging

**Pull request merged**